server/db/models/Product.js

```diff
@@ -0,0 +1,71 @@
+const db = require("../db");
+const { STRING, INTEGER, DECIMAL, TEXT } = require("sequelize");
+
+///////////////// PRODUCT MODEL /////////////////
+const Product = db.define("product", {
+  name: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  // Note: the brand column will be used to pair Product with Brand via script/seed.js.
+  // john - I comment this out because this is automatically create by the association in the indexjs
+  brandName: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  imageURL: {
+    type: STRING,
+    allowNull: false,
+  },
+  price: {
+    type: INTEGER,
```

Comment:
`type: DECIMAL(10,2),`
if that's incorrect- you can achieve it using validator

Reply:
NOTE- decimals with javascript is VERY tricky
```js
console.log(1.4 + 1.2);
```
this is a "floating point error" 
example of how this can boom your project
you have a cart, each cart item is $1.40
you increase the quantity of each cart item, then you sum up the price
fix: toFixed(2)